service-commands/scripts/whitenoise.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generates a random whitenoise mp3 file of given input size.
+ *
+ * Script usage: node whitenoise.js <size> <outFile>
+ */
+
+const { spawn } = require('child_process')
+const ffmpeg = require('ffmpeg-static')
+
+async function run() {
+  try {
+    const { size, outFile } = parseArgs()
+    whitenoise(size, outFile)
+  } catch (err) {
+    console.error(err.message)
+  }
+}
+
+/**
+ * Process two command line args for filesize and filename.
+ *
+ * Defaults to 1 KiB file size, and unix timestamp for filename if args are not provided.
+ */
+function parseArgs() {
+  const args = process.argv.slice(2)
+  const size = args[0] || 1
+  const outFile = args[1] || `whitenoise-${new Date().getTime()}.mp3`
```

Comment:
add option to specify to output path :)

Reply:
Thanks, the second input param (args[1]) can be the filepath as well.

However, this must include the new file name as well. Eg.
```sh
node generateWhiteNoise.js 1000 ~/sample.mp3
```